frontend/www/js/omegaup/components/problem/Edit.vue

```diff
@@ -261,7 +265,7 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
```

Comment:
```suggestion
import { Vue, Component, Prop } from 'vue-property-decorator';
```

esto ya no se está usando aquí.

Reply:
Se va.